TLSharp.Core/TLSharp.Core.csproj

```diff
@@ -42,6 +42,10 @@
     <Reference Include="System.Data" />
     <Reference Include="System.Net.Http" />
     <Reference Include="System.Xml" />
+    <Reference Include="NLog">

```

Comment:
I'm guessing the no-polling feature can exist without adding NLog, let's mix this change in this PR

Reply:
you want NLog removed ?